### PR TITLE
Build ISV campaign page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

*The contact us form will be created in a separate issue so we can get this page out now*
- Build ISV campaign page as per [copydoc](https://docs.google.com/document/d/1pcUx3oJ4ZcgRG6nGv6kzHSf9tFZZnaTK1jzecKFZxGo/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://microk8s-io-513.demos.haus/isv
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copydoc](https://docs.google.com/document/d/1pcUx3oJ4ZcgRG6nGv6kzHSf9tFZZnaTK1jzecKFZxGo/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/master-epics/issues/408

### CODE REVIEW

File: templates/isv.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Build on MicroK8s, the lightweight Kubernetes</h2>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <p>MicroK8s is a zero-ops Kubernetes for delivering your software. Suitable for clouds, edge and IoT clusters, #MicroK8s offers a self-healing high availability feature for enterprise-grade deployments.</p>
```
Comment (minkyngkm) [inline templates/isv.html:74]: "Exceptional" is missing here
```suggestion
      <h3>Exceptional</h3>
      <p>MicroK8s is a zero-ops Kubernetes for delivering your software. Suitable for clouds, edge and IoT clusters, #MicroK8s offers a self-healing high availability feature for enterprise-grade deployments.</p>
```

File: templates/isv.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row">
+    <div class="col-6 u-vertically-center u-align--center">
+      <img src="https://assets.ubuntu.com/v1/f89832c6-Kubernetes-CNCF+compliant.svg" alt="" width="400" height="229">
+    </div>
+    <div class="col-6">
+      <h3>As you want it</h3>
+      <p>With MicroK8s you get a complete, CNCF-compliant Kubernetes environment and all the services you need to run your OCI containers at scale, such as networking &amp; load balancing, storage, service mesh, observability, GPU &amp; FPGA acceleration, multi-cluster management and more - something unique among lightweight Kubernetes distributions.</p>
```
Comment (minkyngkm) [inline templates/isv.html:95]: maybe here is `ndash;` 😉
```suggestion
      <p>With MicroK8s you get a complete, CNCF-compliant Kubernetes environment and all the services you need to run your OCI containers at scale, such as networking &amp; load balancing, storage, service mesh, observability, GPU &amp; FPGA acceleration, multi-cluster management and more &ndash; something unique among lightweight Kubernetes distributions.</p>
```

File: templates/isv.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1>MicroK8s for software developers</h1>
+      <p class="p-headind--2">We deliver containers that you develop</p>
+      <a class="p-button--positive" href="/contact-us">Get in touch</a>
+    </div>
+    <div class="col-4">
+      <img src="https://assets.ubuntu.com/v1/f145e15a-K8s_Multi-cloud_solid.svg" alt="" width="300" height="178">
```
Comment (minkyngkm) [inline templates/isv.html:19]: I can't see the image here on the copy doc.

File: templates/isv.html [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    <div class="col-6">
+      <p>MicroK8s is lightweight, conformant Kubernetes, with sensible defaults that 'just work'. A quick install, easy upgrades and great security make it perfect for micro clouds and edge computing.</p>
+      <h3>Easier deployments</h3>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Faster, appliance-style deployment of infra and apps</li>
+        <li class="p-list__item is-ticked">Reference platform setup simplifies ongoing maintenance</li>
+        <li class="p-list__item is-ticked">Cloud, Kubernetes, edge, infrastructure</li>
+      </ul>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center">
```
Comment (minkyngkm) [inline templates/isv.html:46]:
```suggestion
    <div class="col-6 u-vertically-center u-align--center u-hide--small u-hide--medium">
```

File: templates/isv.html [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+      <img src="https://assets.ubuntu.com/v1/6c0471ea-Kubernetes-Easier+deployments.svg" alt="" width="400" height="229">
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row">
+    <div class="col-6 u-vertically-center u-align--center">
```
Comment (minkyngkm) [inline templates/isv.html:54]:
```suggestion
    <div class="col-6 u-vertically-center u-align--center u-hide--small u-hide--medium">
```

File: templates/isv.html [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+    <div class="col-6">
+      <p>MicroK8s is a zero-ops Kubernetes for delivering your software. Suitable for clouds, edge and IoT clusters, #MicroK8s offers a self-healing high availability feature for enterprise-grade deployments.</p>
+      <h3></h3>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Fire-and-forget K8s with transactional over-the-air updates</li>
+        <li class="p-list__item is-ticked"><a href="https://microk8s.io/high-availability">Self-healing high availability</a></li>
+        <li class="p-list__item is-ticked">GPU acceleration and other add-ons</li>
+      </ul>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center">
```
Comment (minkyngkm) [inline templates/isv.html:82]:
```suggestion
    <div class="col-6 u-vertically-center u-align--center u-hide--small u-hide--medium">
```

File: templates/isv.html [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+      <img src="https://assets.ubuntu.com/v1/d942fae6-Kubernetes-over-air-updates.svg" alt="" width="500" height="131">
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row">
+    <div class="col-6 u-vertically-center u-align--center">
```
Comment (minkyngkm) [inline templates/isv.html:90]:
```suggestion
    <div class="col-6 u-vertically-center u-align--center u-hide--small u-hide--medium">
```

**Review by minkyngkm — COMMENTED** [review]
Looking good Pete! Just need to add the `u-hide--small u-hide--medium` class for images.

And i think what you did for the section colouring is right but it's a bit different from the copy doc so better double checking just in case? :)

File: templates/isv.html (hunk 3 above)
Comment (petesfrench) [reply templates/isv.html:19]: I've just added it in, as it was only mentioned a conversation

**Review by minkyngkm — APPROVED** [review]
LGTM👍

**Pull request merged**